bugbot/rules/lot_of_comments.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SeveralComments(BzCleaner):
```

Comment:
Having the class name consistent with the file name will be better.

Reply:
i am consistent with lot_of_cc and lot_of_vote
I think we should rename all of them in a separate PR